R/quantity.R

```diff
@@ -26,17 +28,40 @@ Quantity <- R6::R6Class(
     },
     quantityType = function(value) {
       private$wrapReadOnlyProperty("QuantityType", value)
+    },
+    formula = function(value) {
+      private$readOnlyProperty("formula", value, private$.formula)
+    },
+    isTable = function(value) {
+      private$readOnlyProperty("isTable", value, self$formula$isTable)
+    },
+    isConstant = function(value) {
+      private$readOnlyProperty("isConstant", value, self$formula$isConstant)
+    },
+    isFormula = function(value) {

```

Comment:
Is  a distributed parameter a formula parameter or a constant parameter... we need to discuss

Reply:
FYI: @Yuri05 
One could also say that at the Quantity Level, a constant is an entity with a formula (constant or distributed) and the a isFormula parmaeter is a quantity with a Table formula or Explicit formula

We can really be has specific or generic as required. I just not sure of the use cases